x-pack/legacy/plugins/actions/server/builtin_action_types/es_index.test.ts

```diff
@@ -19,6 +20,12 @@ import { ActionParamsType, ActionTypeConfigType } from './es_index';
 
 const ACTION_TYPE_ID = '.index';
 const NO_OP_FN = () => {};
+const MOCK_KIBANA_CONFIG_UTILS: ActionsConfigurationUtilities = {
```

Comment:
fyi: more bits to add to the theoretical "testing rig for actions" :-)

Reply:
Agreed